test/tools/jsontests/TransactionTests.cpp

```diff
@@ -185,13 +185,6 @@ class TransactionTestFixture
 	{
 		string const& casename = boost::unit_test::framework::current_test_case().p_name;
 		test::TransactionTestSuite suite;
-
-		if ((casename == "ttWrongRLPFrontier" || casename == "ttWrongRLPHomestead") && test::Options::get().filltests)
-		{
-			suite.copyAllTestsFromFolder(casename);
-			return;
-		}
-
```

Comment:
What happened to these tests that were copied before this PR?

Reply:
the tests now has Copier.json  in  TransactionTestsFiller/ttWrongRLP/*
so when runAllTestsInFolder reads the Copier.json it is now copying such a file instead of filling it. 
WrongRLP tests could not be filled with testeth because it's rlp is malformed

Reply:
Will you document this new behavior in the declaration of `runAllTestsInFolder()`?